Tiltfile

```diff
@@ -8,6 +8,7 @@ version_settings(True, ">=0.22.2")
 settings = {
     "deploy_cert_manager": True,
     "enable_providers": ["docker"],
+    "kind_cluster_name": os.getenv("CAPI_KIND_CLUSTER_NAME", "capi-test"),
```

Comment:
Thx, lgtm in general.


Can you please also add:
```md
**kind_cluster_name** (String, default="kind"): The name of the kind cluster to use when preloading images.
```

to tilt.md again?

I recently dropped it because it wasn't functional anymore.

Reply:
Done! Thanks for catching that.